nibabel/tests/test_analyze.py

```diff
@@ -58,7 +58,7 @@ class TestAnalyzeHeader(tws._TestLabeledWrapStruct):
     header_class = AnalyzeHeader
     example_file = header_file
     sizeof_hdr = AnalyzeHeader.sizeof_hdr
-    supported_np_types = {np.uint8, np.int16, np.int32, np.float32, np.float64, np.complex64}
+    supported_np_types = frozenset((np.uint8, np.int16, np.int32, np.float32, np.float64, np.complex64))
     add_duplicate_types(supported_np_types)
```

Comment:
Right now, `add_duplicate_types` modifies the set in-place, but it could be easily turned into one that accepts a sequence and returns a set. Then you could do:

```
supported np_types = frozenset(expand_duplicate_types((np.uint8, np.int16, np.int32, np.float32, np.float64, np.complex64)))
```

That said, what's the advantage of freezing the set?

Reply:
That's an error. I hadn't realised `supported_np_types` would be later modified.